jax/experimental/jax2tf/tests/correctness_stats.py

```diff
@@ -71,12 +74,15 @@ def _report_failure(error_type: str, msg: str,
   def tf_unimpl(np_dtype: Optional[NpDType] = None,
                 additional_msg: Optional[str] = None,
                 devs: Sequence[str] = all_devices) -> None:
-
-    missing_tf_support = "Missing TF support"
-    msg = "Primitive is unimplemented"
+    msg = "Primitive is unimplemented in TF"
     if additional_msg:
       msg += '; ' + additional_msg
-    _report_failure(missing_tf_support, msg, np_dtype, devs=devs)
+    _report_failure(CATEGORY_MISSING_TF_SUPPORT, msg, np_dtype, devs=devs)
+
+  def tf_maybe_incorrect(np_dtype: Optional[NpDType] = None,
+                         msg: Optional[str] = None,
+                         devs: Sequence[str] = all_devices) -> None:
+    _report_failure(CATEGORY_INCORRECT_RESULTS, msg, np_dtype, devs=devs)
```

Comment:
There is no default message in `tf_maybe_incorrect`, so `msg` should have type `str`!